tests/scale/test_load.py

```diff
@@ -554,14 +556,14 @@ def _create_executor_configuration(service_name: str) -> str:
     Returns: Random name of the new config created.
 
     """
-    mesos_label = "mesos"
+    mesos_label = ""
```

Comment:
why this change?

Reply:
affects MWT load only, currently mwt job runs look for agents with a `mesos` label, out-of-the-box the mesos plugin currently doesn't apply any labels.